drivers/SmartThings/jbl/src/jbl/capability_handler.lua

```diff
@@ -0,0 +1,140 @@
+
+    local _, err, status = conn_info:post_playback(string.format('{"playback": "%s"}', jbl_playback_status))
+    if not err and status == 200 then
+        log.info(string.format("post_playback success, dni = %s", device.device_network_id))
+    elseif status == 404 then
+        log.error("404 error. delete device. dni = " .. tostring(device.device_network_id))
+        device:offline()
+    end
+end
+
+function capability_handler.playback_play_handler(driver, device, args)
+    smartthings_playback_capability_handler(driver, device, "playing")
+end
+
+function capability_handler.playback_pause_handler(driver, device, args)
+    smartthings_playback_capability_handler(driver, device, "paused")
+end
+
+function capability_handler.playback_stop_handler(driver, device, args)
+    smartthings_playback_capability_handler(driver, device, "stopped")
+end
+
+function capability_handler.next_track_handler(driver, device, args)
+    local conn_info = device:get_field(fields.CONN_INFO)
+    log.info(string.format("media_track_control.next_track_handler : dni = %s", device.device_network_id))
+
+    local _, err, status = conn_info:post_playback('{"playback": "next track"}')
+    if err or status == 404 then
+        log.error("media_track_control.next_track_handler : 404 error. delete device. dni = " .. tostring(device.device_network_id))
+        device:offline()
+    end
+end
+
+function capability_handler.previous_track_handler(driver, device, args)
+    local conn_info = device:get_field(fields.CONN_INFO)
+    log.info(string.format("media_track_control.previous_track_handler : dni = %s", device.device_network_id))
+
+    local _, err, status = conn_info:post_playback('{"playback": "previous track"}')
+    if err or status == 404 then
+        log.error("media_track_control.previous_track_handler : 404 error. delete device. dni = " .. tostring(device.device_network_id))
+        device:offline()
+    end
+end
+
+function capability_handler.set_volume_handler(driver, device, args)
+    local volume = args.args.volume
+    local conn_info = device:get_field(fields.CONN_INFO)
+    log.info(string.format("audio_volume.set_volume : dni = %s, volume = %d", device.device_network_id, volume))
+
+
+    local _, err, status = conn_info:post_volume(string.format('{"volume": %d}', volume))
+    if not err and status == 200 then
+        log.info(string.format("post_volume success, dni = %s", device.device_network_id))
+    elseif status == 404 then
+        log.error("set_volume_handler : 404 error. delete device. dni = " .. tostring(device.device_network_id))
+        device:offline()
+    end
+end
+
+function capability_handler.audioNotification_handler(driver, device, args)
+    local uri = args.args.uri
+    local level = args.args.level
+    local conn_info = device:get_field(fields.CONN_INFO)
+
+    log.info(string.format("%s, %s : level = %s", args.capability, args.command, device.device_network_id, level))
+    log.info(string.format("URI: %s", uri))
```

Comment:
nit: these logs are a bit complicated and using %s generates the same as just providing the value so

```
log.info(args.capability, device.device_network_id, "level:", level)
log.info("URI:", uri)
```
is going to give you very similar output w/o having these super long format calls. An alternative would be to use the `%q` format argument which will quote strings and also print `nil` instead `""` which can be helpful sometimes